### PR TITLE
Change workflow buttons

### PR DESCRIPTION
**Pull request by NateWr**
See https://github.com/pkp/pkp-lib/pull/1115

### CODE REVIEW

File: pages/workflow/WorkflowHandler.inc.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ function WorkflowHandler() {
 				'production',
 				'submissionHeader',
 				'submissionProgressBar',
-				'expedite'
+				''
```
Comment (asmecher) [inline pages/workflow/WorkflowHandler.inc.php]: Is this empty string intentional?

File: templates/controllers/modals/submissionMetadata/form/catalogEntrySubmissionReviewForm.tpl [hunk 2]
```diff
@@ -10,29 +10,19 @@
...
-{else}
-	{assign var="formHandlerClass" value="'$.pkp.controllers.form.AjaxFormHandler'"}
-{/if}
+
 <script type="text/javascript">
 	$(function() {ldelim}
 		// Attach the form handler.
 		$('#{$submissionMetadataViewFormId}').pkpHandler(
-			{$formHandlerClass},
+			'$.pkp.controllers.form.AjaxFormHandler',
```
Comment (asmecher) [inline templates/controllers/modals/submissionMetadata/form/catalogEntrySubmissionReviewForm.tpl]: Happy to be removing all the weirdness that this introduced.

File: pages/workflow/WorkflowHandler.inc.php (hunk 1 above)
Comment (NateWr) [inline pages/workflow/WorkflowHandler.inc.php]: Woops! Should be fixed now.

**Comment by asmecher** [comment]
Replaced by https://github.com/pkp/omp/pull/219 (merge conflicts).

**Pull request closed without merging**